docs/modules/user-guide/partials/proc_adding-a-registry-schema.adoc

```diff
@@ -107,45 +165,80 @@ The `index.json` file contains metadata for the stacks and samples in the regist
 | `string`
 | The stack name.
 
-| `version`
-| `string`
-| The stack version.
+| `versions`
+| `version[]`
+| The information of each stack/sample version.
 
 | `attributes`
 | `map[string]apiext.JSON`
 | Map of implementation-dependant free-form YAML attributes.
 
 | `displayName`
 | `string`
-| The display name of a devfile.
+| The display name of a stack.
 
 | `description`
 | `string`
-| The description of a devfile.
+| The description of a stack.
 
 | `type`
 | `DevfileType`
-| The type of a devfile that currently supports stacks and samples.
+| The type: stack/sample.
```

Comment:
To be consisted with all the other phrasing, I would remove the colon. 

Does it make just as much sense to say, `The type of a stack or sample.`